### ISSUE TITLE
[Adaptive Card iOS - DeepFallBack.Json]: Voice over does not announce the 'Dimmed' information for the dimmed text.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8404]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.2" control and select it.
5. Navigate to 'Test' control and activate it.
6. Navigate to 'DeepFallback.Json' and activate it.
7. Navigate to the text having dimmed format and observe the issue.

**Actual Result:**
Voice over does not convey the information as 'Dimmed' for 'Nothing above this' text.

**Expected Result:**
Voice over should convey the descriptive information as 'Dimmed' for 'Nothing above this' text.

**User Experience:**
It will impact the screen reader user if the screen reader does not convey the information as dimmed for the text as user wont be able to know whether text is dimmed or not.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/227840336-d9b5341f-8476-4b76-a647-df757998cb1a.MP4

### Card JSON

```JSON
NA
```
